addons/mail/static/src/components/emoji_list/emoji_list.xml

```diff
@@ -4,7 +4,7 @@
     <t t-name="mail.EmojiList" owl="1">
         <div class="o_EmojiList" t-attf-class="{{ className }}" t-ref="root">
             <t t-foreach="emojis" t-as="emoji" t-key="emoji.unicode">
-                <span class="o_EmojiList_emoji" t-on-click="emojiListView.onClickEmoji" t-att-title="emoji.description" t-att-data-source="emoji.sources[0]" t-att-data-unicode="emoji.unicode">
+                <span class="o_EmojiList_emoji" t-on-click="ev => emojiListView.onClickEmoji(ev)" t-att-title="emoji.description" t-att-data-source="emoji.sources[0]" t-att-data-unicode="emoji.unicode">
```

Comment:
Do we really have to change all of those handlers? It worked just fine with our auto bind.